udaykalra/portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,140 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+:root {
+  --main-color: #fff;
+  --head-color: #000000;
+  --body-color: #121212;
+  --text-color: #e7e7e7;
+  --light-color: #333333;
+}
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+/*Dark theme*/
+html,
+body {
+  margin: 0;
+  height: 100%;
+  background-color: var(--body-color);
+}
+
+body {
+  background: 100%;
+}
+
+div.main-content {
+  background-color: var(--body-color);
+  color: var(--main-color);
+  font-family: 'Roboto', sans-serif;
+  height: 100%;
+}
+
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)),
+    url('hero.jpg');
+
+  height: 50%;
+
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  font-family: 'Roboto', sans-serif;
+  font-size: 40px;
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+}
+
+.hero-text h1 {
+  font-size: 60px;
+}
+footer {
```

Comment:
Style prescribes a newline between rules:
https://google.github.io/styleguide/htmlcssguide.html#Rule_Separation

Also applies to some rules below.

Reply:
This has been implemented.